pkg/resources/common/configmap.go

```diff
@@ -117,6 +117,24 @@ func (r *Reconciler) meshConfig() string {
 	return string(marshaledConfig)
 }
 
+func (r *Reconciler) getLocalityLBConfiguration() *istiov1beta1.LocalityLBConfiguration {
+	var localityLbConfiguration *istiov1beta1.LocalityLBConfiguration
+
+	if !util.PointerToBool(r.Config.Spec.LocalityLB.Enabled) {
```

Comment:
Shouldn't we check first `if r.Config.Spec.LocalityLB != nil`? Because if it is `r.Config.Spec.LocalityLB.Enabled` would fail, isn't it?

Reply:
Not sure actually, but i added a condition, just to be on the safe side.

Reply:
Thanks!